### PR TITLE
Doctrine deprecated functions 11970

### PR DESCRIPTION
**Pull request by lenonleite**
| Q                                      | A
| -------------------------------------- | ---
| Bug fix? (use the a.b branch)          | [ ]
| New feature/enhancement? (use the a.x branch)      | [ ]
| Deprecations?                          | [ ]
| BC breaks? (use the c.x branch)        | [ ]
| Automated tests included?              | [ ] 
| Related user documentation PR URL      | mautic/mautic-documentation#... 
| Related developer documentation PR URL | mautic/developer-documentation#... 
| Issue(s) addressed                     | Fixes #... 

#### Steps to test this PR:

1. Open this PR on Gitpod or pull down for testing locally (see docs on testing PRs [here](https://contribute.mautic.org/contributing-to-mautic/tester))
2.

### CODE REVIEW

**Comment by escopecz** [comment]
@lenonleite is this ready to review or WIP?

File: app/bundles/AssetBundle/Entity/DownloadRepository.php [hunk 1]
```diff
@@ -110,13 +110,13 @@ public function getTopReferrers($query, $limit = 10, $offset = 0)
             ->setMaxResults($limit)
             ->setFirstResult($offset);
 
-        return $query->execute()->fetchAllAssociative();
+        return $query->fetchAllAssociative();
```
Comment (mabumusa1) [inline app/bundles/AssetBundle/Entity/DownloadRepository.php:113]:
```suggestion
        return $query->executeQuery()->fetchAllAssociative();
```

File: app/bundles/AssetBundle/Entity/DownloadRepository.php [hunk 2]
```diff
@@ -129,7 +129,7 @@ public function getHttpStatuses($query)
             ->groupBy('ad.code')
             ->orderBy('count', 'DESC');
 
-        $results = $query->execute()->fetchAllAssociative();
+        $results = $query->fetchAllAssociative();
```
Comment (mabumusa1) [inline app/bundles/AssetBundle/Entity/DownloadRepository.php:132]:
```suggestion
        $results = $query->executeQuery()->fetchAllAssociative();
```

**Review by mabumusa1 — CHANGES_REQUESTED** [review]
There is a couple of places that needs changing other than that, it is good

**Comment by mabumusa1** [comment]
> @lenonleite is this ready to review or WIP?

@escopecz I talked to him in the morning, it is ready for testing

**Comment by RCheesley** [comment]
@lenonleite please provide test instructions to help testers with reviewing this PR? Thanks!

**Comment by RCheesley** [comment]
@lenonleite we need some test instructions here? It'd be good to know what parts it touches and what testers should look for - can you update the description?

**Pull request closed without merging**